tests/integration/test_api_gateway.py

```diff
@@ -127,3 +148,38 @@ def forward_request(self, **kwargs):
 
     # clean up
     proxy.stop()
+
+
+def test_api_gateway_lambda_proxy_integration():
+    test_port = 12123
+    backend_url = 'http://localhost:%s%s' % (test_port, API_PATH_LAMBDA_PROXY_BACKEND)
+
+    # create target HTTP backend
+    class TestListener(ProxyListener):
```

Comment:
I'll have to leave it out of this PR because I have another PR with more functionality that I need to make after this one, and I don't want to convolute it too much. Can we revisit this in a follow-up? The real point of this PR was the behaviour. I'm not exactly sure how to quickly provision everything we'd need to do your test.